README_AWS.md

```diff
@@ -51,7 +51,11 @@ OSX:
 Test The Setup
 --------------
 1. cd openshift-ansible
-1. Try to list all instances:
+1. To create a cluster with one master and two nodes:
```

Comment:
So the point of this section is to test that you have the creds setup right.

I'd prefer a read-only test (like what "./cloud.rb aws list" is).

I think it's a bit of overkill to have to create a cluster in order to test your setup.